components/profile-controller/controllers/profile_controller.go

```diff
@@ -617,16 +621,23 @@ func removeString(slice []string, s string) (result []string) {
 	return
 }
 
-func updateNamespaceLabels(ns *corev1.Namespace) bool {
-	updated := false
+func setNamespaceLabelsFromConfig(ns *corev1.Namespace) {
```

Comment:
nit: maybe we should rename now? because we no longer read config here

Reply:
Sounds good, I have renamed this function.